### PR TITLE
[REVIEW] Update docs to remove references to master [skip ci]

### PR DESCRIPTION
**Pull request by dillon-cullinan**
This PR removes unecessary references to master or updates references to the new main branch in CI scripts and other OPS-owned files.

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: cpp/include/cudf/detail/utilities/hash_functions.cuh [hunk 1]
```diff
@@ -247,7 +247,7 @@ void CUDA_DEVICE_CALLABLE MD5Hash::operator()<string_view>(column_device_view co
 }  // namespace cudf
 
 // MurmurHash3_32 implementation from
-// https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp
+// https://github.com/aappleby/smhasher/blob/main/src/MurmurHash3.cpp
```
Comment (kkraus14) [inline cpp/include/cudf/detail/utilities/hash_functions.cuh:250]:
```suggestion
// https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp
```

File: cpp/libcudf_kafka/include/cudf_kafka/kafka_consumer.hpp [hunk 2]
```diff
@@ -44,7 +44,7 @@ class kafka_consumer : public cudf::io::datasource {
    * operations. This is useful when the need for delayed partition and topic assignment
    * is not known ahead of time and needs to be delayed to as late as possible.
    * Documentation for librdkafka configurations can be found at
-   * https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
+   * https://github.com/edenhill/librdkafka/blob/main/CONFIGURATION.md
```
Comment (kkraus14) [inline cpp/libcudf_kafka/include/cudf_kafka/kafka_consumer.hpp:47]:
```suggestion
   * https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
```

File: cpp/libcudf_kafka/include/cudf_kafka/kafka_consumer.hpp [hunk 3]
```diff
@@ -53,7 +53,7 @@ class kafka_consumer : public cudf::io::datasource {
 
   /**
    * @brief Instantiate a Kafka consumer object. Documentation for librdkafka configurations can be
-   * found at https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
+   * found at https://github.com/edenhill/librdkafka/blob/main/CONFIGURATION.md
```
Comment (kkraus14) [inline cpp/libcudf_kafka/include/cudf_kafka/kafka_consumer.hpp:56]:
```suggestion
   * found at https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
```

File: cpp/src/io/comp/cpu_unbz2.cpp [hunk 4]
```diff
@@ -22,7 +22,7 @@
  *
  * bzip2 license information is available at
  * https://spdx.org/licenses/bzip2-1.0.6.html
- * https://github.com/asimonov-im/bzip2/blob/master/LICENSE
+ * https://github.com/asimonov-im/bzip2/blob/main/LICENSE
```
Comment (kkraus14) [inline cpp/src/io/comp/cpu_unbz2.cpp:25]:
```suggestion
 * https://github.com/asimonov-im/bzip2/blob/master/LICENSE
```

File: cpp/src/io/comp/snap.cu [hunk 5]
```diff
@@ -247,7 +247,7 @@ static __device__ uint32_t Match60(const uint8_t *src1,
 
 /**
  * @brief Snappy compression kernel
- * See http://github.com/google/snappy/blob/master/format_description.txt
+ * See http://github.com/google/snappy/blob/main/format_description.txt
```
Comment (kkraus14) [inline cpp/src/io/comp/snap.cu:250]:
```suggestion
 * See http://github.com/google/snappy/blob/master/format_description.txt
```

File: cpp/src/io/comp/unbz2.h [hunk 6]
```diff
@@ -18,7 +18,7 @@
  *
  * bzip2 license information is available at
  * https://spdx.org/licenses/bzip2-1.0.6.html
- * https://github.com/asimonov-im/bzip2/blob/master/LICENSE
+ * https://github.com/asimonov-im/bzip2/blob/main/LICENSE
```
Comment (kkraus14) [inline cpp/src/io/comp/unbz2.h:21]:
```suggestion
 * https://github.com/asimonov-im/bzip2/blob/master/LICENSE
```

File: cpp/src/io/comp/unsnap.cu [hunk 7]
```diff
@@ -593,7 +593,7 @@ __device__ void snappy_process_symbols(unsnap_state_s *s, int t)
 
 /**
  * @brief Snappy decompression kernel
- * See http://github.com/google/snappy/blob/master/format_description.txt
+ * See http://github.com/google/snappy/blob/main/format_description.txt
```
Comment (kkraus14) [inline cpp/src/io/comp/unsnap.cu:596]:
```suggestion
 * See http://github.com/google/snappy/blob/master/format_description.txt
```

File: cpp/src/io/parquet/page_data.cu [hunk 8]
```diff
@@ -85,7 +85,7 @@ struct page_state_s {
  * @brief Computes a 32-bit hash when given a byte stream and range.
  *
  * MurmurHash3_32 implementation from
- * https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp
+ * https://github.com/aappleby/smhasher/blob/main/src/MurmurHash3.cpp
```
Comment (kkraus14) [inline cpp/src/io/parquet/page_data.cu:88]:
```suggestion
 * https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp
```

File: cpp/src/io/parquet/reader_impl.cu [hunk 9]
```diff
@@ -169,7 +169,7 @@ std::string name_from_path(const std::vector<std::string> &path_in_schema)
   // For the case of lists, we will see a schema that looks like:
   // a.list.element.list.element
   // where each (list.item) pair represents a level of nesting.  According to the parquet spec,
-  // https://github.com/apache/parquet-format/blob/master/LogicalTypes.md
+  // https://github.com/apache/parquet-format/blob/main/LogicalTypes.md
```
Comment (kkraus14) [inline cpp/src/io/parquet/reader_impl.cu:172]:
```suggestion
  // https://github.com/apache/parquet-format/blob/master/LogicalTypes.md
```

File: cpp/src/io/parquet/reader_impl.cu [hunk 10]
```diff
@@ -357,7 +357,7 @@ class aggregate_metadata {
     auto &pfm = per_file_metadata[0];
 
     // see : the "Nested Types" section here
-    // https://github.com/apache/parquet-format/blob/master/LogicalTypes.md
+    // https://github.com/apache/parquet-format/blob/main/LogicalTypes.md
```
Comment (kkraus14) [inline cpp/src/io/parquet/reader_impl.cu:360]:
```suggestion
    // https://github.com/apache/parquet-format/blob/master/LogicalTypes.md
```

File: docs/cudf/source/conf.py [hunk 11]
```diff
@@ -60,8 +60,8 @@
 # source_suffix = ['.rst', '.md']
 source_suffix = {".rst": "restructuredtext", ".md": "markdown"}
 
-# The master toctree document.
-master_doc = "index"
+# The main toctree document.
+main_doc = "index"
```
Comment (kkraus14) [inline docs/cudf/source/conf.py:64]:
```suggestion
# The master toctree document.
master_doc = "index"
```

File: docs/cudf/source/conf.py [hunk 12]
```diff
@@ -157,7 +157,7 @@
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
     (
-        master_doc,
+        main_doc,
```
Comment (kkraus14) [inline docs/cudf/source/conf.py:160]:
```suggestion
        master_doc,
```

File: docs/cudf/source/conf.py [hunk 13]
```diff
@@ -170,7 +170,7 @@
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [(master_doc, "cudf", "cudf Documentation", [author], 1)]
+man_pages = [(main_doc, "cudf", "cudf Documentation", [author], 1)]
```
Comment (kkraus14) [inline docs/cudf/source/conf.py:173]:
```suggestion
man_pages = [(master_doc, "cudf", "cudf Documentation", [author], 1)]
```

File: docs/cudf/source/conf.py [hunk 14]
```diff
@@ -180,7 +180,7 @@
 #  dir menu entry, description, category)
 texinfo_documents = [
     (
-        master_doc,
+        main_doc,
```
Comment (kkraus14) [inline docs/cudf/source/conf.py:183]:
```suggestion
        master_doc,
```

File: python/cudf/cudf/_version.py [hunk 15]
```diff
@@ -200,7 +200,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # refs/heads/ and refs/tags/ prefixes that would let us distinguish
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
-        # "stabilization", as well as "HEAD" and "master".
+        # "stabilization", as well as "HEAD" and "main".
```
Comment (kkraus14) [inline python/cudf/cudf/_version.py:203]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/cudf/cudf/core/dataframe.py [hunk 16]
```diff
@@ -6294,7 +6294,7 @@ def select_dtypes(self, include=None, exclude=None):
         """
 
         # code modified from:
-        # https://github.com/pandas-dev/pandas/blob/master/pandas/core/frame.py#L3196
+        # https://github.com/pandas-dev/pandas/blob/main/pandas/core/frame.py#L3196
```
Comment (kkraus14) [inline python/cudf/cudf/core/dataframe.py:6297]:
```suggestion
        # https://github.com/pandas-dev/pandas/blob/master/pandas/core/frame.py#L3196
```

File: python/cudf/cudf/tests/test_array_function.py [hunk 17]
```diff
@@ -11,7 +11,7 @@
 missing_arrfunc_reason = "NEP-18 support is not available in NumPy"
 
 # Test implementation based on dask array test
-# https://github.com/dask/dask/blob/master/dask/array/tests/test_array_function.py
+# https://github.com/dask/dask/blob/main/dask/array/tests/test_array_function.py
```
Comment (kkraus14) [inline python/cudf/cudf/tests/test_array_function.py:14]:
```suggestion
# https://github.com/dask/dask/blob/master/dask/array/tests/test_array_function.py
```

File: python/cudf/cudf/utils/ioutils.py [hunk 18]
```diff
@@ -864,7 +864,7 @@
 ----------
 kafka_configs : dict, key/value pairs of librdkafka configuration values.
     The complete list of valid configurations can be found at
-    https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
+    https://github.com/edenhill/librdkafka/blob/main/CONFIGURATION.md
```
Comment (kkraus14) [inline python/cudf/cudf/utils/ioutils.py:867]:
```suggestion
    https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
```

File: python/cudf/cudf/utils/utils.py [hunk 19]
```diff
@@ -263,7 +263,7 @@ def get_null_series(size, dtype=np.bool):
 
 
 # taken from dask array
-# https://github.com/dask/dask/blob/master/dask/array/utils.py#L352-L363
+# https://github.com/dask/dask/blob/main/dask/array/utils.py#L352-L363
```
Comment (kkraus14) [inline python/cudf/cudf/utils/utils.py:266]:
```suggestion
# https://github.com/dask/dask/blob/master/dask/array/utils.py#L352-L363
```

File: python/cudf/versioneer.py [hunk 20]
```diff
@@ -14,7 +14,7 @@
 (https://pypip.in/version/versioneer/badge.svg?style=flat)
 ](https://pypi.python.org/pypi/versioneer/)
 * [![Build Status]
-(https://travis-ci.org/warner/python-versioneer.png?branch=master)
+(https://travis-ci.org/warner/python-versioneer.png?branch=main)
```
Comment (kkraus14) [inline python/cudf/versioneer.py:17]:
```suggestion
(https://travis-ci.org/warner/python-versioneer.png?branch=master)
```

File: python/cudf/versioneer.py [hunk 21]
```diff
@@ -175,7 +175,7 @@
 
 * Source trees which contain multiple subprojects, such as
   [Buildbot](https://github.com/buildbot/buildbot), which contains both
-  "master" and "slave" subprojects, each with their own `setup.py`,
+  "main" and "slave" subprojects, each with their own `setup.py`,
```
Comment (kkraus14) [inline python/cudf/versioneer.py:178]:
```suggestion
  "master" and "slave" subprojects, each with their own `setup.py`,
```

File: python/cudf/versioneer.py [hunk 22]
```diff
@@ -623,7 +623,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # refs/heads/ and refs/tags/ prefixes that would let us distinguish
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
-        # "stabilization", as well as "HEAD" and "master".
+        # "stabilization", as well as "HEAD" and "main".
```
Comment (kkraus14) [inline python/cudf/versioneer.py:626]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/cudf/versioneer.py [hunk 23]
```diff
@@ -1015,7 +1015,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
...
```
Comment (kkraus14) [inline python/cudf/versioneer.py:1018]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/cudf_kafka/cudf_kafka/_version.py [hunk 24]
```diff
@@ -200,7 +200,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # refs/heads/ and refs/tags/ prefixes that would let us distinguish
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
-        # "stabilization", as well as "HEAD" and "master".
+        # "stabilization", as well as "HEAD" and "main".
```
Comment (kkraus14) [inline python/cudf_kafka/cudf_kafka/_version.py:203]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/cudf_kafka/versioneer.py [hunk 25]
```diff
@@ -14,7 +14,7 @@
 (https://pypip.in/version/versioneer/badge.svg?style=flat)
 ](https://pypi.python.org/pypi/versioneer/)
 * [![Build Status]
-(https://travis-ci.org/warner/python-versioneer.png?branch=master)
+(https://travis-ci.org/warner/python-versioneer.png?branch=main)
```
Comment (kkraus14) [inline python/cudf_kafka/versioneer.py:17]:
```suggestion
(https://travis-ci.org/warner/python-versioneer.png?branch=master)
```

File: python/cudf_kafka/versioneer.py [hunk 26]
```diff
@@ -175,7 +175,7 @@
 
 * Source trees which contain multiple subprojects, such as
   [Buildbot](https://github.com/buildbot/buildbot), which contains both
-  "master" and "slave" subprojects, each with their own `setup.py`,
+  "main" and "slave" subprojects, each with their own `setup.py`,
```
Comment (kkraus14) [inline python/cudf_kafka/versioneer.py:178]:
```suggestion
  "master" and "slave" subprojects, each with their own `setup.py`,
```

File: python/cudf_kafka/versioneer.py [hunk 27]
```diff
@@ -623,7 +623,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # refs/heads/ and refs/tags/ prefixes that would let us distinguish
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
-        # "stabilization", as well as "HEAD" and "master".
+        # "stabilization", as well as "HEAD" and "main".
```
Comment (kkraus14) [inline python/cudf_kafka/versioneer.py:626]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/cudf_kafka/versioneer.py [hunk 28]
```diff
@@ -1015,7 +1015,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
...
```
Comment (kkraus14) [inline python/cudf_kafka/versioneer.py:1018]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/custreamz/custreamz/_version.py [hunk 29]
```diff
@@ -200,7 +200,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # refs/heads/ and refs/tags/ prefixes that would let us distinguish
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
-        # "stabilization", as well as "HEAD" and "master".
+        # "stabilization", as well as "HEAD" and "main".
```
Comment (kkraus14) [inline python/custreamz/custreamz/_version.py:203]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/custreamz/custreamz/kafka.py [hunk 30]
```diff
@@ -21,7 +21,7 @@ def __init__(self, kafka_configs):
             Dict of Key/Value pairs of librdkafka
             configuration values. Full list of valid configuration
             options can be found at
-            https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
+            https://github.com/edenhill/librdkafka/blob/main/CONFIGURATION.md
```
Comment (kkraus14) [inline python/custreamz/custreamz/kafka.py:24]:
```suggestion
            https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
```

File: python/custreamz/custreamz/kafka.py [hunk 31]
```diff
@@ -67,7 +67,7 @@ def __init__(self, kafka_configs):
...
```
Comment (kkraus14) [inline python/custreamz/custreamz/kafka.py:70]:
```suggestion
            https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
```

File: python/custreamz/versioneer.py [hunk 32]
```diff
@@ -14,7 +14,7 @@
 (https://pypip.in/version/versioneer/badge.svg?style=flat)
 ](https://pypi.python.org/pypi/versioneer/)
 * [![Build Status]
-(https://travis-ci.org/warner/python-versioneer.png?branch=master)
+(https://travis-ci.org/warner/python-versioneer.png?branch=main)
```
Comment (kkraus14) [inline python/custreamz/versioneer.py:17]:
```suggestion
(https://travis-ci.org/warner/python-versioneer.png?branch=master)
```

File: python/custreamz/versioneer.py [hunk 33]
```diff
@@ -175,7 +175,7 @@
 
 * Source trees which contain multiple subprojects, such as
   [Buildbot](https://github.com/buildbot/buildbot), which contains both
-  "master" and "slave" subprojects, each with their own `setup.py`,
+  "main" and "slave" subprojects, each with their own `setup.py`,
```
Comment (kkraus14) [inline python/custreamz/versioneer.py:178]:
```suggestion
  "master" and "slave" subprojects, each with their own `setup.py`,
```

File: python/custreamz/versioneer.py [hunk 34]
```diff
@@ -623,7 +623,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # refs/heads/ and refs/tags/ prefixes that would let us distinguish
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
-        # "stabilization", as well as "HEAD" and "master".
+        # "stabilization", as well as "HEAD" and "main".
```
Comment (kkraus14) [inline python/custreamz/versioneer.py:626]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/custreamz/versioneer.py [hunk 35]
```diff
@@ -1015,7 +1015,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
...
```
Comment (kkraus14) [inline python/custreamz/versioneer.py:1018]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/dask_cudf/dask_cudf/_version.py [hunk 36]
```diff
@@ -200,7 +200,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # refs/heads/ and refs/tags/ prefixes that would let us distinguish
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
-        # "stabilization", as well as "HEAD" and "master".
+        # "stabilization", as well as "HEAD" and "main".
```
Comment (kkraus14) [inline python/dask_cudf/dask_cudf/_version.py:203]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/dask_cudf/versioneer.py [hunk 37]
```diff
@@ -14,7 +14,7 @@
 (https://pypip.in/version/versioneer/badge.svg?style=flat)
 ](https://pypi.python.org/pypi/versioneer/)
 * [![Build Status]
-(https://travis-ci.org/warner/python-versioneer.png?branch=master)
+(https://travis-ci.org/warner/python-versioneer.png?branch=main)
```
Comment (kkraus14) [inline python/dask_cudf/versioneer.py:17]:
```suggestion
(https://travis-ci.org/warner/python-versioneer.png?branch=master)
```

File: python/dask_cudf/versioneer.py [hunk 38]
```diff
@@ -175,7 +175,7 @@
 
 * Source trees which contain multiple subprojects, such as
   [Buildbot](https://github.com/buildbot/buildbot), which contains both
-  "master" and "slave" subprojects, each with their own `setup.py`,
+  "main" and "slave" subprojects, each with their own `setup.py`,
```
Comment (kkraus14) [inline python/dask_cudf/versioneer.py:178]:
```suggestion
  "master" and "slave" subprojects, each with their own `setup.py`,
```

File: python/dask_cudf/versioneer.py [hunk 39]
```diff
@@ -623,7 +623,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # refs/heads/ and refs/tags/ prefixes that would let us distinguish
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
-        # "stabilization", as well as "HEAD" and "master".
+        # "stabilization", as well as "HEAD" and "main".
```
Comment (kkraus14) [inline python/dask_cudf/versioneer.py:626]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

File: python/dask_cudf/versioneer.py [hunk 40]
```diff
@@ -1015,7 +1015,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
...
```
Comment (kkraus14) [inline python/dask_cudf/versioneer.py:1018]:
```suggestion
        # "stabilization", as well as "HEAD" and "master".
```

**Pull request merged**